environment.yml

```diff
@@ -9,17 +9,17 @@ dependencies:
   - cartopy>=0.18
   - compilers
   - gdal
-  - esmpy
+  - esmpy>=8.0.1
   - esmvalcore>=2.2.0,<2.3
   - iris>=3.0.1
   - matplotlib>3.3.1  # bug in 3.3.1 but 3.3.3 gets installed automatically
   # Non-Python dependencies
   - cdo>=1.9.7
   - eccodes!=2.19.0  # cdo dependency; something messed up with libeccodes.so
   - imagemagick
-  - mpich<3.4  # avoid 3.4.1 external_2 from conda-forge like the plague
-  - nco
-  - ruamel.yaml
+  # - mpich<3.4  # avoid 3.4.1 external_2 from conda-forge like the plague; stable 3.3.2 OK
```

Comment:
I'd like to keep that for now, just in case conda decides to install it again (note that conda installs 3.3.2 now so chances are in the near future it might jump to 3.4.1, which build hash, dunno, it's a lottery, but if the unlucky `external_2` creeps up again we'll know what's the bugger)